internal/cmd/auth.go

```diff
@@ -159,13 +210,17 @@ func authenticationGRPC(
 				rgxs = append(rgxs, rgx)
 			}
 
-			interceptors = append(interceptors, authmiddlewaregrpc.EmailMatchingInterceptor(logger, rgxs, authOpts...))
+			interceptors = append(interceptors, selector.UnaryServerInterceptor(authmiddlewaregrpc.EmailMatchingInterceptor(logger, rgxs, authOpts...), authmiddlewaregrpc.ClientTokenInterceptorSelector()))
 		}
 
 		if authCfg.Methods.Token.Enabled {
-			interceptors = append(interceptors, authmiddlewaregrpc.NamespaceMatchingInterceptor(logger, authOpts...))
+			interceptors = append(interceptors, selector.UnaryServerInterceptor(authmiddlewaregrpc.NamespaceMatchingInterceptor(logger, authOpts...), authmiddlewaregrpc.ClientTokenInterceptorSelector()))
```

Comment:
Are these going to stack and cause Flipt to check multiple times when multiple instances are enabled?

Reply:
Discussed offline, this is just my misinterpretation of why the selectors are being added in these additional place.
The interceptor is only being registered once.